### ISSUE TITLE
Other A11y: Many Pages: The screen automatically closes after selecting an option

### ISSUE THREAD

**Issue by elisenwazuoke** [Expensify/App#77375]
## Action Performed:
Precondition: The User should be logged in

1. Open the Expensify app
2. Tap the Account button
3. Tap the Preferences button
4. Tap any option to select it
5. Note that the screen automatically closes after selecting an option

Other Occurrences:
1. On Preferences - Priority Mode
2. On Profile - Country
3. On Profile - Timezone

## Expected Result:
Selecting an option should highlight the choice but keep the user on the current screen, allowing them to verify their selection, continue making choices, or press a dedicated "Confirm" button to close the screen.

## Actual Result:
Immediately after a user taps an option, the entire view disappears, abruptly moving the user to the next screen without confirmation or context.

## Area issue was found in:
Preferences and  Subsections

## Failed WCAG checkpoints
WCAG 3.2.2 On Input
https://www.w3.org/WAI/WCAG22/Understanding/on-input

## User impact:
This helps users with disabilities by making interactive content more predictable. Unexpected changes of context can be disorienting for users with visual disabilities or cognitive limitations. Individuals who are blind or have low vision may have difficulty knowing when a visual context change has occurred, such as a new window popping up. In this case, warning users of context changes in advance minimizes confusion when the user discovers that the back button no longer behaves as expected. Some individuals with low vision, with reading and intellectual disabilities, and others who have difficulty interpreting visual cues may benefit from additional cues in order to detect changes of context.

## Suggested resolution:
Ensure the screen doesn ot close automatically after selecting an option. Add a button for the user to confirm the selection.
https://www.w3.org/WAI/WCAG22/Understanding/on-input

## Environment(s)
Samsung Galaxy S24 Ultra - Android 14, Partner - Native (No mobile browser)
Apple iPhone 11 - iOS 17.5.1, H0T Mobile - Native (No mobile browser)

## Screenshots/Videos
![Image](https://github.com/user-attachments/assets/cb6214ee-667c-442d-ba1a-b4e9843884c1)

https://github.com/user-attachments/assets/e50d6749-1eae-4027-aee1-2465db15ca13

**Version Number:** Production
**Reproducible in staging?:** NA
**Reproducible in production?:** Yes
**Email or phone of affected tester (no customers):** NA
**Logs:** https://stackoverflow.com/c/expensify/questions/4856 NA
**Notes/Photos/Videos:** Any additional supporting documentation Yes
**Expensify/Expensify Issue URL:** https://platform.applause.com/company/12102/products/36628/community-issues/7006337
**Issue reported by:** NA
**Slack conversation:** NA

**Comment by rushatgabhane** [comment]
This issue was retested in **BFV cycle 544085 (Native Apps)** on 2026-04-28 and is **Not Fixed**.

### Screenshots / Videos
- https://platform.applause.com/services/links/v1/external/dd0711914a2f342b2e6e4eb0b89565cd90a4232257e08b27f234acb74ac873eb

### Retest notes
The main issue is not fixed

Note: The other occurrences are not fixed

Description: After a user selects an option, the screen closes automatically and immediately transitions to the next view. This abrupt change occurs without any confirmation or contextual feedback, which may confuse users and disrupt their interaction flow.

Issue verified on Apple iPhone 16 - iOS 26.2.1, Wi-Fi - Native 
App Version: Version 9.3.63 and App Name: Expensify
Please see the attached video

**Comment by rushatgabhane** [comment]
confirmed with QA that this is not an issue https://expensify.slack.com/archives/C9YU7BX5M/p1772214722722529?thread_ts=1761671288.437029&cid=C9YU7BX5M
